build.py

```diff
@@ -172,11 +172,29 @@ def main():
 
             # Unpack chromium tarball
             get_logger().info('Unpacking chromium tarball...')
-            downloads.unpack_downloads(download_info, downloads_cache, source_tree, extractors)
+            downloads.unpack_downloads(download_info, downloads_cache, source_tree, None, extractors)
         else:
             # Clone sources
             subprocess.run([sys.executable, str(Path('ungoogled-chromium', 'utils', 'clone.py')), '-o', 'build\src', '-p', 'win32' if args.x86 else 'win64'], check=True)
```

Comment:
We might want to escape the backslash in `'build\\src'`. 

Also, we could bump the Python version requirement to 3.12. It builds and runs fine on my end. 

And yeah I'm good with dropping tarball support if the tarball requires more work.

Reply:
You are right, I fixed that.